src/hooks/useNodeJsContributorsApi.tsx

```diff
@@ -1,7 +1,12 @@
+/* eslint-disable camelcase */
 import { useEffect, useState } from 'react';
+import type {
+  Contributor,
+  ContributorApiResponse,
+} from '../types/nodejsContributors';
 
-const CONTRIBUTORS_API_URI =
-  'https://api.github.com/repos/nodejs/node/contributors?per_page=1';
+const limitContributors = 5;
```

Comment:
If this is meant to be a constant it should be in all caps
```suggestion
const LIMIT_CONTRIBUTORS = 5;
```